src/Client.cpp

```diff
@@ -150,6 +150,13 @@ err_code_t Client::version(types::broadcast_result_t** results, size_t* nHosts)
 }
 
 
+err_code_t Client::quit() {
+  broadcastCommand(keywords::kQUIT, 4);
```

Comment:
The inner interface([`BufferWriter::takeBuffer(const char* const buf, size_t buf_len)`](https://github.com/douban/libmc/blob/9ed101bcde3556adbb5a1228bcf2805ddd8e8129/src/BufferWriter.cpp#L38)) is designed to be compatible with arbitrary string buffer.  (e.g.: if `\0` is in a buffer, we can never call `strlen` to get the actual length.)